src/plugins/dna_export/src/ExportMSA2MSADialog.h

```diff
@@ -35,9 +35,16 @@ class ExportMSA2MSADialog : public QDialog, Ui_ExportMSA2MSADialog {
 public:
     ExportMSA2MSADialog(const QString &defaultFileName, const DocumentFormatId &f, bool wholeAlignmentOnly, QWidget *p);
 
+    enum class UnknownAmino {
```

Comment:
Please document any new types/methods/fields.

I would suggest to use "above the subject" doc style like /** some text */  because it will never go to the line below and automatically shown by IDEs. 

The suffix-style comments like 
int fieldX; // fieldX is lorem ipsum...
are good only if they are short (single line) and are never shown by IDE as contextual docs. I think they are good for the inner comments, but not for the top-level once.

Reply:
Done